src/command-line/utils/prompt.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const log = require("../../log");
+const read = require("read");
+
+module.exports = (options, callback) => {
+	options.prompt = log.prompt(options.text);
```

Comment:
`prompt` != `text` on purpose for 2 reasons:
- We could still use `prompt` if we want to define a tailor-made CLI prompt without the `log.prompt` (we'll need to do `... = options.prompt || ...` or something though)
- `prompt({ prompt: "..." }, ...);` feels weird to me :D

Stupid? 😅